cypress/e2e/group3/deletion.ts

```diff
@@ -13,75 +13,99 @@
+}
 
-  function goToPrivatePage(entry: Entry): void {
-    cy.visit('/');
-    cy.visit(`${entry.myPrefix}/${entry.path}`);
-    cy.contains(entry.path);
-  }
+function checkCanDeleteNeverPublished(entry: Entry) {
+  unpublicize(entry);
+  goToPrivatePage(entry);
+  cy.contains('button', 'Delete').should('be.visible').should('not.be.disabled').click();
+  cy.get('[data-cy=delete-no]').should('be.visible').should('not.be.disabled').click();
+  goToPrivatePage(entry);
+  cy.contains(entry.path).should('exist');
+  cy.contains('button', 'Delete').should('be.visible').should('not.be.disabled').click();
+  cy.get('[data-cy=delete-yes]').should('be.visible').should('not.be.disabled').click();
+  cy.wait(1000);
+  cy.contains(entry.path).should('not.exist');
+}
 
+function deletionTests(entry: Entry) {
   it('Should not be able to delete an entry that is published', () => {
-    entries.forEach((entry) => {
-      unpublicize(entry);
-      goToPrivatePage(entry);
-      cy.contains('button', 'Delete').should('be.visible').should('not.be.disabled');
-      cy.contains('button', 'Publish').should('be.visible').should('not.be.disabled').click();
-      goToPrivatePage(entry);
-      cy.contains('button', 'Delete').should('not.exist');
-    });
+    checkCantDeleteIsPublished(entry);
   });
 
   it('Should not be able to delete an entry that was previously published', () => {
-    entries.forEach((entry) => {
-      unpublicize(entry);
-      goToPrivatePage(entry);
-      cy.contains('button', 'Delete').should('be.visible').should('not.be.disabled');
-      cy.contains('button', 'Publish').should('be.visible').should('not.be.disabled').click();
-      goToPrivatePage(entry);
-      cy.contains('button', 'Unpublish').should('be.visible').should('not.be.disabled').click();
-      goToPrivatePage(entry);
-      cy.contains('button', 'Delete').should('not.exist');
-    });
+    checkCantDeletePreviouslyPublished(entry);
   });
 
   it('Should be able to delete an entry that was never published', () => {
-    entries.forEach((entry) => {
-      unpublicize(entry);
-      goToPrivatePage(entry);
-      cy.contains('button', 'Delete').should('be.visible').should('not.be.disabled').click();
-      cy.get('[data-cy=delete-no]').should('be.visible').should('not.be.disabled').click();
-      goToPrivatePage(entry);
-      cy.contains(entry.path).should('exist');
-      cy.contains('button', 'Delete').should('be.visible').should('not.be.disabled').click();
-      cy.get('[data-cy=delete-yes]').should('be.visible').should('not.be.disabled').click();
-      cy.wait(1000);
-      cy.contains(entry.path).should('not.exist');
-    });
+    checkCanDeleteNeverPublished(entry);
   });
+}
+
+describe('Entry Deletion 0', () => {
+  init();
+  deletionTests(entries[0]);
+});
+
+describe('Entry Deletion 1', () => {
+  init();
+  deletionTests(entries[1]);
+});
+
+describe('Entry Deletion 2', () => {
+  init();
+  deletionTests(entries[2]);
```

Comment:
Since we're not iterating through `entries` anymore, we could break out each `Entry` into its own variable (`workflow`, `notebook`, etc) and describe the tests likewise.

```
const notebook = {... }
// ...
describe('Notebook Deletion'), () => {
  init();
  deletionTests(notebook);
}
```